lib/worldwide/address.rb

```diff
@@ -148,7 +148,9 @@ def valid?
     end
 
     def concatenate_address1
-      additional_fields = region.combined_address_format["address1"] || []
+      default_address_format = region.combined_address_format.dig("default", "address1") || []
+      additional_fields_string = default_address_format.map { |field| send(field["key"].underscore.to_sym) }.join(" ")
```

Comment:
This is a little clunky - I didn't want to directly pass street name and street num to `detect_script`,  but we don't technically know what the relevant additional fields are in this point in the code. So, I decided to grab the default ones.